brainwidemap/decoding/functions/process_motors.py

```diff
@@ -0,0 +1,386 @@
+from one.api import ONE
+from brainbox.io.one import load_channel_locations 
+import brainbox.behavior.wheel as wh
+from brainbox.processing import bincount2D
+from ibllib.atlas import regions_from_allen_csv
+import ibllib.atlas as atlas
+from ibllib.atlas import AllenAtlas
+from ibllib.atlas.regions import BrainRegions
+from brainbox.io.one import SpikeSortingLoader
+import numpy as np
+from pathlib import Path
+from collections import Counter
+import math
+import matplotlib.pyplot as plt
+from matplotlib.lines import Line2D
+from scipy.stats import pearsonr, spearmanr
+from copy import deepcopy
+import pandas as pd
+import random
+import seaborn as sns
+import matplotlib as mpl
+from sklearn.decomposition import PCA
+from sklearn.manifold import MDS
+from scipy.stats import zscore
+import itertools
+from mpl_toolkits.mplot3d import Axes3D
+import os, sys
+from scipy.interpolate import interp1d
+import matplotlib
+from scipy import stats
+from scipy.stats import percentileofscore
+
+
+import glob
+from braindelphi.params import CACHE_PATH
+from datetime import datetime
+import pickle
+
+matplotlib.rcParams.update({'font.size': 10})
+
+one = ONE()
+ba = AllenAtlas()
+br = BrainRegions()
+T_BIN = 0.02
+Fs = {'left':60,'right':150, 'body':30}
+
+# specify binning type, either bins or sampling rate; 
+
+sr = {'licking':'T_BIN','whisking_l':'T_BIN', 'whisking_r':'T_BIN', 
+      'wheeling':'T_BIN','nose_pos':'T_BIN', 'paw_pos_r':'T_BIN', 
+      'paw_pos_l':'T_BIN'}
+
+blue_left = [0.13850039, 0.41331206, 0.74052025]
+red_right = [0.66080672, 0.21526712, 0.23069468]
+cdi = {0.8:blue_left,0.2:red_right,0.5:'g',-1:'cyan',1:'orange'}
+def blockPrint():
+    sys.stdout = open(os.devnull, 'w')
+# Restore
+def enablePrint():
+    sys.stdout = sys.__stdout__
+
+def get_all_sess_with_ME():
+    one = ONE()
+    # get all bwm sessions with dlc
+    all_sess = one.alyx.rest('sessions', 'list', 
+                              project='ibl_neuropixel_brainwide_01',
+                              task_protocol="ephys", 
+                              dataset_types='camera.ROIMotionEnergy')
+    eids = [s['url'].split('/')[-1] for s in all_sess]
+    
+    return eids  
+
+
+def find_nearest(array,value):
+    idx = np.searchsorted(array, value, side="left")
+    if idx > 0 and (idx == len(array) or math.fabs(value - array[idx-1]) < math.fabs(value - array[idx])):
+        return idx-1
+    else:
+        return idx
+
+def get_licks(XYs):
```

Comment:
This function is only used in preprocess_motors_old which is itself no longer used. Remove